modules/mcagar/CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## Unreleased
 
+### Added
+- `getSel()` to the `EditorTypes.Selection` interface
+- `setRawSelection()` to the `TinySelections` helper methods
```

Comment:
```suggestion
- The `getSel()` function to the `EditorTypes.Selection` interface.
- The `setRawSelection()` function to the `TinySelections` helper methods to allow setting the selection without using any TinyMCE APIs.
```
Just some suggestions, up to you if you use them. Also changelog entries normally end with a Jira number or a full stop 🤷  This one seems to be using fullstops instead of Jira numbers, so I've added that.

Reply:
yeah, I'm not super attached to internal changelogs customers will never see